backends/cuda/ceed-cuda-operator.c

```diff
@@ -150,6 +150,9 @@ static int CeedOperatorSetupFields_Cuda(CeedQFunction qf, CeedOperator op,
         ierr = CeedElemRestrictionCreateVector(Erestrict, NULL,
                                                &evecs[i + starte]);
         CeedChkBackend(ierr);
+        // Allocate array
+        ierr = CeedVectorSetArray(evecs[i + starte], CEED_MEM_DEVICE, CEED_COPY_VALUES,
+                                  NULL); CeedChkBackend(ierr);
```

Comment:
`CeedElemRestriction` needs to use `GetArray` instead of `GetArrayWrite` because the convention is that we sum into the target for transpose mode, so we need to use initialized arrays for restrictions.

Reply:
Ah, that makes sense -- but each backend already takes care of that inside those functions/kernels, no?  Since they're only for the transpose mode, and we don't want to unnecessarily initialize. Also, `CEED_COPY_VALUES` and `NULL` just allocates memory instead of initializing, so we're still not getting an initialized array here.

Reply:
Woops, wrong word. We can't use `GetArrayWrite` inside of the restriction because of the summing convention, but this allocation is for the inputs, which use the NoTranspose mode, so we overwrite all values. 

Transpode mode : need initialized values
NoTransposed mode : need allocated memory

Reply:
Backends could choose `GetArray` for Transpose mode and `GetArrayWrite` for NoTranspose mode though...

Reply:
True.  That makes a good case for using `SetArray` to allocate (but not initialize) here.  👍